src/components/autosuggest-text/autossugest-text.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { SuggestionsFromQueryStringQuery } from "../../core/dbc-gateway/generated/graphql";
+
+export interface AutosuggestTextProps {
+  responseData: SuggestionsFromQueryStringQuery["suggest"]["result"];
+  currentQ: string;
+}
+
+export const AutosuggestText: React.FC<AutosuggestTextProps> = ({
+  responseData
+}) => {
+  return (
+    <>
+      {responseData.map((item) => {
+        return (
+          <li className="autosuggest-text-container__item text-body-medium-regular">
+            {item.__typename === "Creator" ? `${item.name} (forfatter)` : null}
```

Comment:
It could be an idea to create an AutosuggestTextItem component that contains all this logic. It would be a cleaner look here and be more declarative ;)

Reply:
done :)